readme.md

```diff
@@ -31,10 +31,10 @@ $ ncc build input.js -o dist
 Outputs the build of `input.js` into `dist/index.js`.
 
 ```bash
-$ ncc run input.js
+$ ncc run input.js --source-map
```

Comment:
This is a redundant example right? Since `run` will have it by default.

Per my comment below, I'm also considering whether `build` will have it as well, and we'll just go with `--no-source-map`.